protons/driver.py

```diff
@@ -660,10 +665,6 @@ def _update_forces(self, titration_group_index, final_titration_state_index, ini
                     force.setExceptionParameters(
                         exc['exception_index'], exc['particle1'], exc['particle2'], exc['chargeProd'], exc['sigma'], exc['epsilon'])
 
-            # Update parameters in Context, if specified.
```

Comment:
It might be useful to add a note to the docstring of this function, as a reminder that this part is now separate.

Something along the lines of:

```
Notes
------
* Please ensure that the context is updated after calling this function, by using `force.updateParametersInContext(context)` for each force that has been updated. 
```